packages/gatsby-source-nacelle/src/source-nodes/index.js

```diff
@@ -147,21 +128,16 @@ async function fetchRemoteImageNodes(dataType, node, gatsbyApi) {
   const isImage = (nodeMediaEntry) =>
     nodeMediaEntry &&
     nodeMediaEntry.type &&
-    nodeMediaEntry.type.startsWith('image');
-  if (dataType === 'product') {
+    nodeMediaEntry.type.toLowerCase().startsWith('image');
+  if (dataType === 'Product') {
     await createRemoteImageFileNode(
       node,
-      ['featuredMedia', 'media'],
+      [
+        ['content', 'featuredMedia'],
+        ['content', 'media']
+      ],
       gatsbyApi,
       { isImage }
     );
-  } else if (dataType === 'collection') {
-    await createRemoteImageFileNode(node, 'featuredMedia', gatsbyApi, {
-      isImage
-    });
-  } else if (dataType === 'content') {
-    await createRemoteImageFileNode(node, 'featuredMedia', gatsbyApi, {
-      isImage
-    });
```

Comment:
@jasonredick I just noticed this - is this an intentional change? If so, how are we handling non-`product` node image file creation?

Reply:
@NWRichmond From what I can tell, there are no fields for media in collections (product or content) or content, unless it's buried within the `fields` JSON, which I don't believe we can work with here.

Reply:
I agree. We can revisit this when we aren't representing those `fields` as `JSON`. Thanks!!!